docs/general/getting_started.md

```diff
@@ -15,6 +15,12 @@ And then execute:
 ```
 $ bundle
 ```
+### Available Commands
```

Comment:
@beauby  What you think about this point  below?

when new user are reading the documentation , I think him needs is all the commands available after the that him  can do with these command.